resources_pages/install_ds_stack_windows.md

```diff
@@ -334,14 +334,14 @@ we need to use the `pip` package manager instead of `conda`:
 pip install https://github.com/jupyter/nbconvert/archive/6.0.0a6.zip
 ```
 
-> Note: we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
+> **Note**: we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
 
 ### JupyterLab setup
 
 We will be using the Jupytext Python package and the JupyterLab git extension to facilitate using Jupyter notebooks with Git & GitHub. Install them via the following commands:
 
 ```
-conda install -y nodejs=10.*
+conda install -y nodejs=15.*
```

Comment:
Could you change this the same way we changed  it for Mac in armands PR? And also add it to the linux doc?

Reply:
This phrase has not an example now, as we are not using `conda install -y`, but it is OK to keep it I guess.

> `conda` will show you the packages that will be downloaded,
> and you can press enter to proceed with the installation.
> If you want to answer `yes` by default and skip this confirmation step,
> you can replace `conda install` with `conda install -y`.

Reply:
I will follow up on this separately if needed